sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/postgres/PostgresInt8Tests.java

```diff
@@ -581,13 +575,13 @@ public void testINT64MINOverflow() {
         );
     }
 
-    @Test @Ignore("Integer wrapping: https://github.com/feldera/feldera/issues/1186")
+    @Test @Ignore("fails for Calcite optimized version")
```

Comment:
are these in the right file? These do not look like int8 tests

Reply:
Yes. They were taken from [here](https://github.com/postgres/postgres/blob/master/src/test/regress/sql/int8.sql#L200).